basic-algorithm-scripting/mutations.js

```diff
@@ -3,7 +3,7 @@ function mutation(arr) {
   let secondWord = arr[1].toLowerCase();
 
   for (let i = 0; i < secondWord.length; i++) {
```

Comment:
Please change the style of this loop.... instead of use index i please use "for...of" structure using char as the name of control variable.
One point here... Ever use "for...of" structure when you can. It's the best most intuitive way to describe a loop.